src/server/scripts/Northrend/FrozenHalls/ForgeOfSouls/forge_of_souls.cpp

```diff
@@ -28,7 +28,13 @@ enum Yells
     SAY_SYLVANAS_INTRO_6                        = 5,
 
     SAY_JAINA_OUTRO                             = 0,
-    SAY_SYLVANAS_OUTRO                          = 0,
+    SAY_SYLVANAS_OUTRO                          = 0
+};
+
+enum Gossip
```

Comment:
Use unique names so that there are no conflicts in the future.

Reply:
You mean the enum?

Reply:
Thanks for the suggestion, changes applied.